StationeersLaunchPad/LaunchPadAlertGUI.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Assets.Scripts;
+using ImGuiNET;
+using System;
+using System.Collections.Generic;
+using UI.ImGuiUi;
+using UnityEngine;
+
+namespace StationeersLaunchPad {
+  public static class LaunchPadAlertGUI {
+    public static bool IsActive;
+    public static string Title;
+    public static string Description;
+
+    public static Dictionary<string, Func<bool>> Buttons = new();
+
+    public static void DrawPreload() {
+      if (GameManager.IsBatchMode)
+        return;
+
+      DrawAlert();
+    }
+
+    public static void Show(string title, string description, Dictionary<string, Func<bool>> buttons) {
```

Comment:
could we do an array of tuples instead of a dictionary here? dictionary iteration order is undefined so people could get the buttons in a random order.

doing something like:
```cs
public static void Show(string, title, string description, params (string, Func<bool>)[] buttons)
```
would let you call it like this instead and keep the order:
```cs
LaunchPadAlertGui.Show("title", "description",
  ("action1", () => { return true; }),
  ("action2", () => { return false; })
);
```